sdk/appservice/azure-resourcemanager-appservice/pom.xml

```diff
@@ -89,7 +89,7 @@
     <dependency>
       <groupId>com.azure.resourcemanager</groupId>
       <artifactId>azure-resourcemanager-appcontainers</artifactId>
-      <version>1.2.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure.resourcemanager:azure-resourcemanager-appcontainers;dependency} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;beta_com.azure.resourcemanager:azure-resourcemanager-appcontainers;dependency} -->
```

Comment:
We need to depend on the beta of appcontainers lib in the test of appservice.

The recording contains APIs in appcontainers, hence the "build from source" tests would be same as this released bata.